### PR TITLE
Admin grid groups revision

### PR DESCRIPTION
**Pull request by guerler**
Removes grid and form makos used for the admin grid group handling. Branched from #4385.

### CODE REVIEW

File: lib/galaxy/webapps/galaxy/controllers/admin.py [hunk 1]
```diff
@@ -1055,254 +1059,137 @@ def create_role( self, trans, **kwd ):
...
+                if group in [ x.group for x in role.groups ]:
+                    in_groups.append( trans.security.encode_id( group.id ) )
+                all_groups.append( ( group.name, trans.security.encode_id( group.id ) ) )
+            return { 'title'  : 'Role \'%s\'' % role.name,
+                     'message': 'Role \'%s\' is currently associated with %d user(s) and %d group(s).' %
+                                ( role.name, len( in_users ), len( in_groups ) ),
+                     'status' : 'info',
+                     'inputs' : [ build_select_input( 'groups', 'Groups', all_groups, in_groups ),
+                                  build_select_input( 'users', 'Users', all_users, in_users ) ] }
+            return { 'message' : 'Not showing associated datasets, there are too many.', 'info' : 'info' }
```
Comment (mvdbeek) [inline lib/galaxy/webapps/galaxy/controllers/admin.py]: This line is unreachable.

Comment (guerler) [reply lib/galaxy/webapps/galaxy/controllers/admin.py]: Thanks for catching that @mvdbeek.

**Review by martenson — APPROVED** [review]
I think the select/unselect all (roles/users) feature does not make much sense in this context, and neither does 'save settings' button, when creating a group.

I assume these changes are in place because of the toolformication of these forms, which is planned to be temporary so I guess I am ok with this even though I would prefer to not have these slightly confusing changes in the UI.

**Comment by guerler** [comment]
Thanks @martenson. Changing the title and/or hiding the select all field is easy to do. I will add it in a follow up.

**Pull request merged**